Modules/Microsoft365DSC/Examples/Resources/IntuneFirewallRulesPolicyWindows10ConfigMgr/3-Remove.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+This example removes a Device Control Policy.
```

Comment:
This description looks incorrect

Reply:
It sure is... That's what I get from copy pasting 😆